app/assets/javascripts/deploy_groups.js

```diff
@@ -0,0 +1,17 @@
+// get back a list of deploy groups, generate a select list with 'em
+// these are being stored in the DOM to avoid having to make an ajax query etc
+
+var populateProjectSelectList = function() {
+  var environment = $("#secret_environment_permalink").val()
+  var groups = JSON.parse($('#deployGroupList').attr('imbededData'))
```

Comment:
can also dump the data on the page for easy access ... https://github.com/zendesk/samson/blob/master/plugins/kubernetes/app/views/admin/kubernetes/deploy_group_roles/_form.html.erb#L45-L67